src/routes/onboarding.js

```diff
@@ -51,6 +51,8 @@ const OnboardingStack = StackNavigator(
     Tabs: {
       screen: Tabs,
     },
+    // BUG HERE: After onboarding, if user logs out then they get taken to intro screen
+    // but they can't go to the auth screen unless they restart the app
```

Comment:
@edencakir any idea how to fix this?

Reply:
This one is caused because of our separated navigation stacks. Unfortunately, I couldn't follow the latest patches of `react-navigation` library, and nested navigation was an unresolved issue back then.

> This one is caused because of our separated navigation stacks.
Like if we had all our screens in a single stack, that'd make it possible to jump from one to another without any issues.

Reply:
We can look into this later i guess